src/init/features/firestore.ts

```diff
@@ -159,16 +157,16 @@ var _getIndexesFromConsole = function(projectId) {
         original: e,
       });
     });
-};
+}
 
-module.exports = function(setup, config) {
+export async function doSetup(setup: any, config: any): Promise<any> {
   setup.config.firestore = {};
 
-  return requireAccess({ project: setup.projectId }, [scopes.CLOUD_PLATFORM])
-    .then(function() {
+  return requireAccess({ project: setup.projectId }) // TODO(rpb): what is this: [scopes.CLOUD_PLATFORM])
```

Comment:
I think it's an artifact of a refactor that wasn't removed. I think you can safely remove it.

Reply:
kk